### DISCUSSION TITLE
Not able to install @openzeppelin into VS code

### DISCUSSION THREAD

**Discussion by mustafaassaad** [ConsenSys-archive/truffle#5509] [Installation]
I am following the “build an ERC-4907 rentable NFT” https://youtu.be/aiGVzcX0P6U

I have typed “Npm i @openzeppelin/contracts” in my terminal on VS code and it’s failing to integrate it with my code, I don’t know why, I have followed everything in the tutorial.
![Uploading image.jpg…]()

**Comment by FlaviuWeb3** [comment]
Hi @mustafaassaad could you please run again npm i -D @openzeppelin/contracts and share with me the output for this command?

**Reply by mustafaassaad** [reply]
![image](https://user-images.githubusercontent.com/89585671/188967625-a6a8283c-2306-4616-8a47-c125debc5854.jpeg)


Not sure why it's not downloading, in the tutorial at 23:00 it downloads smoothly.

**Reply by kevinbluer** [reply]
Hey @mustafaassaad from what I can tell from the above screenshot it looks like it did install correctly...the messages (which may be a bit different from what you saw in the tutorial) are from the `npm audit` which [automatically runs](https://docs.npmjs.com/auditing-package-dependencies-for-security-vulnerabilities#running-a-security-audit-with-npm-audit) after an `npm i`.
